packages/client/src/router/index.js

```diff
@@ -103,6 +103,23 @@ export const routes = [
           }
         },
       },
+      {
+        path: '/my-grants/assigned',
+        name: 'assigned',
+        redirect: shareTerminologyEnabled() ? '/my-grants/shared-with-your-team' : undefined,
```

Comment:
It's a bit more DRY to use the named route here  — `{ name: 'shared-with-your-team' }` — instead of the raw route string